extras/update_hashes.py

```diff
@@ -0,0 +1,15 @@
+import glob
+import os, os.path as osp
+import subprocess
+
+path = osp.split(__file__)[0]
+target = "jslibs"
+
+subprocess.call(["python", osp.join(path, "jslibfetcher.py"), target])
+subprocess.call(["python", osp.join(path, "build_hashes.py"), target])
+
+for file in glob.glob(osp.join(path, target, "hashes-*.txt")):
+	os.rename(file,
+	  osp.join(path, osp.pardir, "validator", "testcases", osp.split(file)[1]))
```

Comment:
It looks like you can do [`os.path.basename(file)`](https://docs.python.org/2/library/os.path.html#os.path.basename) instead of `os.path.split(file)[1]`.